easybuild/easyconfigs/l/libreadline/libreadline-7.0-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'libreadline'
+version = '7.0'
+
+homepage = 'http://cnswww.cns.cwru.edu/php/chet/readline/rltop.html'
+description = """The GNU Readline library provides a set of functions for use by applications that
+ allow users to edit command lines as they are typed in. Both Emacs and vi editing modes are available.
+ The Readline library includes additional functions to maintain a list of previously-entered command lines,
+ to recall and perhaps reedit those lines, and perform csh-like history expansion on previous commands."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
+toolchainopts = {'pic': True}
+
+sources = ['readline-%(version)s.tar.gz']
+source_urls = ['http://ftp.gnu.org/gnu/readline']
+
+patches = ['libreadline-%(version)s-bugfix.patch']
+
+builddependencies = [('binutils', '2.27')]
+dependencies = [('ncurses', '6.0')]
+
+# for the termcap symbols, use EB ncurses
+preconfigopts = "env LDFLAGS='-lncurses'"
```

Comment:
only if the `ncurses` we're using here provides a separate libtinfo, which it doesn't (only the one built with `dummy` does)

We discussed this during yesterday's conf call, @vanzod is going to come back to #4049, but it should only relate to the `dummy` ncurses (which was being picked up even though the `GCCcore` one should be used, for some reason)